Xbmc expects an older version of automake, this patch by bangkok_manouel fixes it.
---
--- xbmc/lib/libdvd/libdvdcss/configure.ac.orig	2013-01-05 18:33:55.822355943 +0700
+++ xbmc/lib/libdvd/libdvdcss/configure.ac	2013-01-05 18:34:45.072355845 +0700
@@ -5,7 +5,7 @@
 AC_CANONICAL_SYSTEM
 
 AM_INIT_AUTOMAKE(libdvdcss, 1.2.10)
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADER(config.h)
 
 AC_PROG_CC
 AC_STDC_HEADERS
